src/layers/arc-layer/arc-layer.ts

```diff
@@ -100,7 +101,13 @@ export const arcColumnLabels = {
   lng1: 'arc.lng1'
 };
 
-export const arcVisConfigs = {
+export const arcVisConfigs: {
```

Comment:
why do we need to type this?

Reply:
In base layer we have function with this type of param:
`registerVisConfig(layerVisConfigs: {
    [key: string]: keyof LayerVisConfigSettings | ValueOf<LayerVisConfigSettings>;
  }) `
Without type we have error like `Type 'string' is not assignable to type 'keyof LayerVisConfigSettings | ValueOf<LayerVisConfigSettings>'`